codegen/src/main/java/software/amazon/awssdk/codegen/poet/builder/AsyncClientBuilderClass.java

```diff
@@ -49,6 +50,9 @@ public AsyncClientBuilderClass(IntermediateModel model) {
         this.builderClassName = ClassName.get(basePackage, model.getMetadata().getAsyncBuilder());
         this.builderBaseClassName = ClassName.get(basePackage, model.getMetadata().getBaseBuilder());
         this.endpointRulesSpecUtils = new EndpointRulesSpecUtils(model);
+        this.serviceConfigClassName = PoetUtils.classNameFromFqcn(model.getMetadata().getFullInternalPackageName()
+                                                                  + "." + model.getMetadata().getServiceName()
+                                                                  + "ServiceClientConfiguration");
```

Comment:
Minor: looks like we have the same code in other places;we should probably create a method in PoetExtension to reuse it so that if we want to change it (although unlikely), we can just change it in one place.

Reply:
added method in PoetExtension